mirdata/jams_utils.py

```diff
@@ -117,6 +117,20 @@ def jams_converter(
                 )
             jam.annotations.append(chords_to_jams(chords))
 
+
+    # notes
+    if note_data is not None:
+        if type(note_data) != list:
+            raise TypeError('note_data should be a list of tuples')
+        for notes in note_data:
+            if type(notes) != tuple:
+                raise TypeError(
+                    'note_data should be a list of tuples, '
+                    + 'but is a list of something else'
```

Comment:
would be nice to say : "but contains a `type(notes)` element`

Reply:
Sorry I forgot about this one! I don't understand the comment, I guess it could be a list of anything, including  `type(note)`?

Reply:
@magdalenafuentes lists in Python may contain elements of mixed types.
`[1, 2.0, 'a', (3, 4)]` is perfectly valid as a list.

This is why the phrasing of "X is a list of something else" is a bit misleading. For all we know, some elements in `note_data` may be tuples while others may be non-tuples. From the user's perspective, it's informative to know what sort of non-tuple is in there. The error will be triggered at the first non-tuple encountered.

Does that make sense?